public/icons/Calendar.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="26" height="24" viewBox="0 0 26 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
for the SVGs, best to grab the layer named `Icon` in figma as they have a consistent 30px square box. the actual icons themselves vary in size so by ensuring they're all in a 30px box we don't have to worry about spacing or sizing
<img width="163" alt="Screen Shot 2021-01-18 at 10 55 35 pm" src="https://user-images.githubusercontent.com/19167953/104912369-492a7e00-59e0-11eb-9b7f-a023ecbcf75e.png">